src/GCRealTimeMon/Program.cs

```diff
@@ -180,7 +180,12 @@ static async Task<Configuration.Configuration> GetConfiguration(Options options)
             if (string.IsNullOrEmpty(configurationFile))
             {
                 // the default .yaml file is at the same location as the CLI global tool / console application
-                configurationFile = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "DefaultConfig.yaml");
+                var location = Assembly.GetExecutingAssembly().Location;
+                // single file mode doesn't support getting the location of an assembly
+                if (string.IsNullOrEmpty(location))
+                    configurationFile = "DefaultConfig.yaml";
+                else
+                    configurationFile = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "DefaultConfig.yaml");
```

Comment:
you don't need to get `Assembly.GetExecutingAssembly().Location` again, do you?

Reply:
Done.